net/src/net.cpp

```diff
@@ -89,6 +90,322 @@ int getAsync(lua_State* L)
     return lua_yield(L, 0);
 }
 
+static std::unordered_map<int, std::shared_ptr<uWS::App>> serverInstances;
```

Comment:
Can we actually do `std::unique_ptr` here and plain borrows (`uWS:App*`) in `ServerLoopState`? This avoids any atomics, and I think the memory model here is sane.